### PR TITLE
DAV-3234: Updated the code base to make more agnostic. Had to change the requests python lib

### PR DESCRIPTION
**Pull request by jwbanning**
Updated the code base to make more agnostic. Had to change the requests python lib

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+To see what Product Sentiment Insights API is, please refer to the Bazaarvoice [documentation](https://developer.bazaarvoice.com/product-sentiment-insights-api/home) for details.
+
 ## License
 This project is licensed under the Apache 2.0 License - see the [LICENSE](LICENSE) file for details.
 
 ## Reference Tools
-The Displayable Content Export is an HTTP-protocol based API, and as such, is not bound to any specific programming language. Your application might be written in Java, javascript, Ruby, Python, Perl, C#, C++, or any other programming language in which HTTP requests can be made over the internet.
-
-The reference languages contained in this project are simple examples of how one might interact with the Displayable Content Export. These examples are not meant for production use. Your implementation will require exception handling, HTTP retries in case of network failures, etc.
+Bazaarvoice export APIs are HTTP-protocol based API, and as such, is not bound to any specific programming language. Your application might be written in Java, javascript, Ruby, Python, Perl, C#, C++, or any other programming language in which HTTP requests can be made over the internet.
```
Comment (frederickf) [inline README.MD]: "Bazaarvoice export APIs are HTTP-protocol based API, and as such, is not bound to any specific programming language."

Mixed plural and singular the that first part is a bit repetitive. How about "Bazaarvoice export APIs based on the HTTP-protocol, and as such, are not bound to any specific programming language."

File: code/export-downloader/README.MD [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Reference Languages
+[Python 2.7](python)
+
+[Node.js](node)
+
+## Common Prerequisites
+For each of the reference tools, the following prerequisites apply:
+
+1. You should have a file called "config.json" under "./exporter-downloader" path, which includes code snippet below. Please replace "{EXPORT-PRODUCT-PASSKEY}" and "{EXPORT-PRODUCT-SECRET}" with the passkey and secret key strings provided by the Bazaarvoice team.
```
Comment (frederickf) [inline code/export-downloader/README.MD]: MIssing "the": "which includes _the_ code snippet below."

File: code/export-downloader/node/README.MD [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# DCE Downloader Tools in Node.js
+# Export Downloader Tools in Node.js
 
-Two Node.js scripts files are provided to get data from the Bazaarvoice Displayable Content Export service. Please refer to [documentation](https://developer.bazaarvoice.com/displayable-content-export) for details.
+Two Node.js scripts files are provided to get data from the Bazaarvoice Export APIs . Please refer to [documentation](https://developer.bazaarvoice.com/poducts) for details.
```
Comment (frederickf) [inline code/export-downloader/node/README.MD]: Missing "the": "Please refer to _the_ [documentation..."

File: README.MD (hunk 1 above)
Comment (brijendra98) [inline README.MD]:
```suggestion
Bazaarvoice export APIs are HTTP-protocol based API, and as such, is not bound to any specific programming language. Your application might be written in Java, JavaScript, Ruby, Python, Perl, C#, C++, or any other programming language in which HTTP requests can be made over the internet.
```

Comment (jwbanning) [reply README.MD]: fixed with Frederick's suggestion

**Review by zach — APPROVED** [review]

**Review by brijendra98 — APPROVED** [review]

**Review by arnm — APPROVED** [review]

**Pull request merged**